src/main/java/lotto/LotteryTicket.java

```diff
@@ -0,0 +1,40 @@
+package lotto;
+
+import java.util.Collections;
+import java.util.List;
+
+public class LotteryTicket {
+    private final List<Integer> lottoTicket;
+
+    public LotteryTicket(List<Integer> autoPickedLottoNumber) {
+        Collections.sort(autoPickedLottoNumber);
+        this.lottoTicket = autoPickedLottoNumber;
+    }
+
+    public static LotteryTicket createAutoLotteryNumber(List<Integer> autoPickedLottoNumber) {
+        return new LotteryTicket(autoPickedLottoNumber);
+    }
+
+    public int size() {
+        return lottoTicket.size();
+    }
+
+    public int countMatch(WinningNumber winningNumber) {
+        int cnt = 0;
+        for (String no : winningNumber.getWinningNumber()){
```

Comment:
```suggestion
        for (String no : winningNumber.getWinningNumber()) {
```

https://github.com/next-step/java-lotto-pro/pull/179#discussion_r744207385 코멘트를 확인해주세요!